default_provider: "qwen" # openai qwen

providers:
  openai:
    api:
      base_url: "https://api.openai.com/v1"
      # key_env_var: "OPENAI_API_KEY"
      # key_path: "/home/maintenance/Code/instruction/assistant_ws_v2/src/assistant_robot/config/keys/openai_key.txt"
      key_path: "/home/diana/Code/assistant_ws_v2/src/assistant_robot/config/keys/openai_key.txt"
      timeout: 30
      max_retries: 3
      default_model: "gpt-4o"  #gpt-4o  gpt-4   
      models:
        - name: "gpt-4o"
          default_temperature: 0.7
        - name: "gpt-4"
          default_temperature: 0.2
        - name: "gpt-3.5-turbo"
          default_temperature: 0.1
    # 模板映射，同 prompt_manager 所需
    # templates_path: "/home/maintenance/Code/instruction/assistant_ws_v2/src/assistant_robot/config/prompts/"
    templates_path: "/home/diana/Code/assistant_ws_v2/src/assistant_robot/config/prompts/"
    templates:      
      llm_intent: "intent_prompt.j2"
      llm_qa: "qa_prompt.j2"
      llm_clarifier: "clarifier_prompt.j2"      
      llm_planner: "planner_prompt.j2"
      #rag_query: "rag_prompt.j2"
      chat: "chat_prompt.j2"

  qwen:
    api:
      base_url: "https://dashscope.aliyuncs.com/compatible-mode/v1"
      key_path: "/home/maintenance/Code/instruction/assistant_ws_v2/src/assistant_robot/config/keys/qwen_key.txt"
      # key_path: "/home/diana/Code/assistant_ws_v2/src/assistant_robot/config/keys/qwen_key.txt"
      # 或者用环境变量（可选）
      # key_env_var: "QWEN_API_KEY"
      timeout: 30
      max_retries: 3
      default_model: "qwen-flash"  # qwen-flash  qwen-plus qwen3-max deepseek-r1  Moonshot-Kimi-K2-Instruct  llama-4-maverick-17b-128e-instruct
      models:
        - name: "qwen-plus"
          default_temperature: 0.7
        - name: "qwen-flash"
          default_temperature: 0.6
        - name: "qwen-turbo"
          default_temperature: 0.5
    templates_path: "/home/maintenance/Code/instruction/assistant_ws_v2/src/assistant_robot/config/prompts/"
    # templates_path: "/home/diana/Code/assistant_ws_v2/src/assistant_robot/config/prompts/"
    templates:
      llm_intent: "intent_prompt_QWen.yaml"
      # chat: "chat_prompt_QWen.yaml"
      llm_qa: "qa_prompt_QWen.yaml"
      llm_clarifier: "clarifier_prompt_QWen.yaml"
      llm_planner: "planner_prompt_QWen.yaml"


